packages/caliper-ethereum/lib/ethereum-connector.js

```diff
@@ -67,7 +67,7 @@ class EthereumConnector extends ConnectorBase {
         if (!ethereumConfig.url) {
             throw new Error(
                 'No URL given to access the Ethereum SUT. Please check your network configuration. ' +
-                'Please see https://hyperledger.github.io/caliper/v0.3/ethereum-config/ for more info.'
+                'Please see https://hyperledger-caliper.github.io/caliper/v0.3/ethereum-config/ for more info.'
```

Comment:
The code has a reference to an old version of the docs ? If you could just delete this line and the plus sign on the previous line that would be great.